workflow/controller/controller.go

```diff
@@ -217,10 +217,15 @@ func (wfc *WorkflowController) Run(ctx context.Context, wfWorkers, workflowTTLWo
 	}
 	logCtx := log.WithField("id", nodeID)
 
+	instanceID := "default-instance-id"
+	if wfc.Config.InstanceID != "" {
+		instanceID = wfc.Config.InstanceID
+	}
+
```

Comment:
I would suggest. like. this.
```golang
  leaderName := "workflow-controller"
	if wfc.Config.InstanceID != "" {
		leaderName = fmt.Sprintf("%s-%s",  leaderName, wfc.Config.InstanceID)
	}
```

Reply:
Yup that is clean, will do that in another PR.